assignments/arrays.js

```diff
@@ -63,32 +63,53 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car 33 is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}` );
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
+console.log(`${inventory[inventory.length - 1].car_make}, ${inventory[inventory.length - 1].car_model}`);
 
 // ==== Challenge 3 ====
 // The marketing team wants the car models listed alphabetically on the website. Sort all the car model names into alphabetical order and log the results in the console
-let carModels = [];
-console.log();
+let carModels = function(){
+    sortedModels = []
+    for (i = 0; i < inventory.length; i++) {
+    sortedModels.unshift(inventory[i].car_model);
+} 
+return sortedModels.sort();
+}
+console.log(carModels());
 
 // ==== Challenge 4 ====
 // The accounting team needs all the years from every car on the lot. Create a new array from the dealer data containing only the car years and log the result in the console.
 let carYears = [];
-console.log();
+for (i = 0; i < inventory.length; i++) {
+    carYears.unshift(inventory[i].car_year);
+} 
+console.log(carYears);
 
 // ==== Challenge 5 ====
 // The car lot manager needs to find out how many cars are older than the year 2000. Using the carYears array you just created, find out how many cars were made before the year 2000 by populating the array oldCars and logging it's length.
 let oldCars = [];
-console.log(); 
+for (i = 0; i < carYears.length; i++) {
+    if (carYears[i] > 2000)
+    oldCars.push(carYears[i]);
+}
+console.log(oldCars.length); 
 
 // ==== Challenge 6 ====
 // A buyer is interested in seeing only BMW and Audi cars within the inventory.  Return an array that only contains BMW and Audi cars.  Once you have populated the BMWAndAudi array, use JSON.stringify() to show the results of the array in the console.
 let BMWAndAudi = [];
-console.log();
+for (i = 0; i < inventory.length; i++) {
+    if (inventory[i].car_make == 'BMW') {
```

Comment:
```suggestion
    if (inventory[i].car_make == 'BMW' || inventory[i].car_make === "Audi") {
```
You could use the or operator rather than an else if statement, but the way you have it is also fine. If you change the line like I am suggesting though you'd also want to take out the entire else if block below